### PR TITLE
Fix navigation on Market Live page

### PR DESCRIPTION
**Pull request by fenglish**
[NOPS-168](https://financialtimes.atlassian.net/browse/NOPS-168)

The navigation contents is displayed on the bottom and never be hidden.
When click the navigation icon on the header, just jump to the bottom.

**The content should be displayed in a drawer like other alphaville pages do.**
 
It was because o-header javascript was not set for the page.

ftalphaville ft com_marketslive

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -34,7 +34,9 @@ alphavilleBuildTools(gulp, {
 		{
 			id: 'ml-index',
 			standalone: 'mlIndexBundle',
+			js: './assets/js/marketLive.js',
 			sass: './assets/scss/ml-index.scss',
+			'build-js': 'ml-Index.js',
```
Comment (binaryberry) [inline gulpfile.js:39]: is that capital "I" in Index intentional?

Comment (fenglish) [reply gulpfile.js:39]: Oh no. It is not. I will change it. Thank you.

File: views/ml-index.handlebars [hunk 2]
```diff
@@ -1,5 +1,13 @@
 {{#contentFor "head"}}
 	<link rel="stylesheet" href="{{assetsBasePath}}/build/ml-index.css" />
+	<script>
+		(function () {
+			ctmLoadScript({
+				src: "{{assetsBasePath}}/build/ml-Index.js",
```
Comment (binaryberry) [inline views/ml-index.handlebars:6]: @fenglish There's also one there!

Comment (binaryberry) [reply views/ml-index.handlebars:6]: Pointing out typos gives me such joy.

Comment (fenglish) [reply views/ml-index.handlebars]: 😄

**Comment by fenglish** [comment]
It looks like JS file name is camel case and CSS file name is with hyphens.
So I changed the file names changed to follow with the rule.

e.g. 
```js
		{
			id: 'meetTheTeam',
			standalone: 'meetTheTeam',
			js: './assets/js/meetTheTeam.js',
			sass: './assets/scss/meet-the-team.scss',
			'build-js': 'meetTheTeam.js',
			'build-css': 'meet-the-team.css'
		},
```

**Pull request merged**